### DISCUSSION TITLE
When is the estimated time of stable version 7 being released?

### DISCUSSION THREAD

**Discussion by mnrendra** [storybookjs/storybook#21231] [Help]
Hi, may I know when is the estimated time of **stable** version 7 being released? Thank you!

**Comment by CasperSocio** [comment]
If you have a look at the [Burndown board](https://github.com/orgs/storybookjs/projects/8) for version 7, you can see that there's currently 25 tasks in the general availability (GA) column that needs to be resolved before 7.0.0 is released. As the beta continues to uncover more requirements, the list of tasks for GA might get longer and prolong the development time. But as the list gets shorter, you can start looking forward to the stable release 🥳
